### PR TITLE
Fixed `$value` variable not updating

### PR DESCRIPTION
**Pull request by JavierLeon9966**

### CODE REVIEW

**Comment by SOF3** [comment]
Alright, so this is a bug, but any chance it breaks existing code?

**Comment by JavierLeon9966** [comment]
Currently I don’t know any plugins that might use `NAN`, `INF` or `-INF` with libasynql, but this could occur for example if they get the value from a SQL query and pass it as a parameter, it will throw an error because it expected a float

**Review by SOF3 — APPROVED** [review]
This is BC breaking, but it was a bug so I consider it no need to bump major. Also I doubt anyone relies on the strings anyway.

File: libasynql/src/poggit/libasynql/sqlite3/Sqlite3Thread.php [hunk 1]
```diff
@@ -111,7 +111,8 @@ protected function executeQuery($sqlite, int $mode, string $query, array $params
 				$colInfo = [];
 				$rows = [];
 				while(is_array($row = $result->fetchArray(SQLITE3_ASSOC))){
-					foreach(array_values($row) as $i => &$value){
+					$row = array_values($row);
+					foreach($row as $i => &$value){
```
Comment (SOF3) [inline libasynql/src/poggit/libasynql/sqlite3/Sqlite3Thread.php:115]: This caused SQLite to not preserve keys anymore!

**Pull request merged**